### PR TITLE
Allow specifying module content inline

### PR DESCRIPTION
**Pull request by lightoze**

### CODE REVIEW

File: manifests/module.pp [hunk 1]
```diff
@@ -44,7 +47,7 @@
     require ::selinux::refpolicy_package
   }
 
-  if ($builder == 'simple' and $source_if != undef) {
+  if ($builder == 'simple' and ($source_if != undef or $content_if != undef)) {
```
Comment (bastelfreak) [inline manifests/module.pp]: This should do the same:

```
if ($builder == 'simple' and ($source_if or $content_if)) 
```

Comment (lightoze) [inline manifests/module.pp]: Not exactly (think empty string). Anyway, I just copied original code style. Do you think it must be changed?

**Comment by vinzent** [comment]
can you add some spec tests?

did you run the beaker tests?

**Comment by lightoze** [comment]
Beaker tests revealed that pick_default from stdlib does not work as I expected - somehow $content_if_real was not undef for `selinux_module_refpolicy_spec.rb` resource `puppet_test_a`. All fixed now.

**Comment by oranenj** [comment]
blah, how does this review thing work...

File: manifests/module.pp (hunk 1 above)
Comment (oranenj) [inline manifests/module.pp]: @lightoze  I like the more compact style now that there is more than one parameter, but it doesn't matter that much. Fix it if you like. :)

**Comment by lightoze** [comment]
Added more documentation, also made a fix in existing example (wrong 'builder' attribute).

**Comment by oranenj** [comment]
Great. Thanks!

I could use this feature myself, so I'll merge this as is and do some testing later.

**Pull request merged**